index.html

```diff
@@ -382,5 +382,78 @@ <h3>
         </pre>
       </section>
     </section>
+    <section id='ABNF' class=informative>
+      <h2>ABNF</h2>
+      <p>
+        The Augmented Backus-Naur Form (ABNF) rules provided below are non-normative. 
+	If there is a discrepancy between requirements in the normative sections and the ABNF, the requirements in the normative sections prevail.
+      </p>
+      <p>
+	Some definiations use the core rules as defined in [[RFC5234]] <a href="https://tools.ietf.org/html/rfc5234#appendix-B.1">appendix B.1</a>. 
+      </p>
+      <p>
+	For addr-spec see [[RFC5322]], <a href="https://tools.ietf.org/html/rfc5322#section-3.4.1">section 3.4.1</a>.
+      </p>
+      <p>
+	For absolute-URI see [[RFC3986]], <a href="https://tools.ietf.org/html/rfc3986#section-4.3">section 4.3</a>.
+      </p>
+
+      <p>
+        <section id='BagitProfileJsonfileABNF'>
+        <h3>Bagit Profile Json file</h3>
+        </section>
+        <pre>
+        profile                     = "{" profile-info bag-info manifests-required manifests-allowed fetch-allowed serialization serialization-types tag-manifests tags-allowed tags-required bagit-version "}"
+        profile-info                = DQUOTE "BagIt-Profile-Info" DQUOTE ":{" profile-identifier profile-bagit-version profile-organization-name profile-name profile-email profile-description profile-version "},"
+        profile-identifier          = DQUOTE "BagIt-Profile-Identifier" DQUOTE ":" DQUOTE absolute-URI DQUOTE ","
+        profile-bagit-version       = DQUOTE "BagIt-Profile-Version" DQUOTE ":" version ","
+        profile-organization-name   = DQUOTE "Source-Organization" DQUOTE ":" value ","
+        profile-name                = DQUOTE "Contact-Name" DQUOTE ":" value ","
+        profile-email               = DQUOTE "Contact-Email" DQUOTE ":" email-address ","
+        email-address               = DQUOTE addr-spec DQUOTE
+        profile-description         = DQUOTE "External-Description" DQUOTE ":" value ","
+        profile-version             = DQUOTE "Version" DQUOTE ":" version ","
+        version                     = DQUOTE 1*DIGIT "." 1*DIGIT DQUOTE
```

Comment:
`version` rule supports multiple other rules with varying numbers of dots in their version numbers. A more generalized version production would be. Might considering calling it "dotted-version" to allow for more generalized (undotted) version strings 
```
version = DQUOTE 1*DIGIT *("." 1*DIGIT) DQUOTE